### PR TITLE
shippit

### PR DESCRIPTION
**Pull request by e6voe9**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,4 @@ don't let a developer hear these words
 - contract
 - 🫶
 - regression
+- safari is good
```
Comment (Hacksore) [inline README.md]: so fucking based!

**Comment by maxdemaio** [comment]
This

**Comment by maxdemaio** [comment]
@e6voe9 if you just synch with the main branch you should be good

**Comment by samyarkd** [comment]
lgtm

**Pull request merged**